### ISSUE TITLE
All test outputs on the custom data set are HBB

### ISSUE THREAD

**Issue by HolleworldJIN** [yijingru/BBAVectors-Oriented-Object-Detection#108]
Hi, Thank you for your works.
I trained your network on custom dataset (the training dataset is about 200, only one object per image with one category), and I found that all the test outputs are HBB, but the traing image and testing image have rotated object.
I checked the training dataset and it's annotation is correct.
In addtion, I observed that the main part of loss is the 'wh' loss, about 3~5.
Do you have any suggestions to fix the problem?

Some Info:
1. 4 * GTX3080Ti
2. batch_size=20
3. epoch=200
4. image shape is (width:2446, height:2048, channel:3)

**Comment by yijingru** [comment]
You may want to check ground-truth for Orientation`alpha`
https://github.com/yijingru/BBAVectors-Oriented-Object-Detection/blob/7efd410f5f4ded94aca986aeb0bd1292235d1314/datasets/base.py#L166
